scrapy/core/downloader/tls.py

```diff
@@ -48,6 +48,11 @@ def _identityVerifyingInfoCallback(self, connection, where, ret):
                         'Remote certificate is not valid for hostname "{}"; {}'.format(
                             self._hostnameASCII, e))
 
+                except ValueError as e:
+                    logger.warning(
+                        'SSL/TLS verification failed for hostname "{}"; {}'.format(
+                            self._hostnameASCII, e))
```

Comment:
It looks like the problem is that ip address is not a hostname, and Twisted passes it to [verify_hostname](https://service-identity.readthedocs.io/en/stable/api.html#service_identity.pyopenssl.verify_hostname). 

So verification is failed, but it is not failed because SSL/TLS algorithm tells it should fail, but arguably because of a twisted bug (or service_identity inflexibility).